core/src/main/java/com/sequenceiq/cloudbreak/structuredevent/converter/StackToStackDetailsConverter.java

```diff
@@ -42,6 +42,8 @@ public StackDetails convert(Stack source) {
         StackDetails stackDetails = new StackDetails();
         stackDetails.setId(source.getId());
         stackDetails.setName(source.getName());
+        stackDetails.setType(source.getType().name());
```

Comment:
In the current json under cluster we already have a ClusterType, which has values of (HDP*|HDF*)
we are now introducing another type field. Can we name this property differently? 
`
"cluster": {
    "id": 88,
     ...

    "clusterType": "HDP-2.6",
    "clusterVersion": "2.6.5.0-292",
    ...
    "defaultFileSystem": null
  }
`